Balancer-Bots/Swap-FlashLoan-Fee-Change/src/agent.spec.ts

```diff
@@ -24,8 +24,8 @@ const IRRELEVANT_IFACE = new ethers.utils.Interface(["event IrrelevantEvent()"])
 
 export function createFlashLoanFeeFinding(newFee: string): Finding {
   return Finding.from({
-    name: `FlashLoan fee changed`,
-    description: `A FlashLoan fee percentage change was detected`,
+    name: "FlashLoan fee changed from the Balancer 'ProtocolFeesCollector' contract",
```

Comment:
```suggestion
    name: "FlashLoan fee changed on the Balancer 'ProtocolFeesCollector' contract",
```
🤔??

Reply:
hmm possible. normally there are 2 FlashLoan/Swap fees, one is set via the pools, and the other one is set by the governance which is this protocol feel collector contract. `on` sounds better.